_build/templates/default/sass/login.scss

```
@@ -183,3 +180,45 @@ a {
   border: 1px solid #aaa;
   position: relative;
 }
+
+@media (max-width: 640px) {
+    #container {
+        margin: 50px auto 0;
+        width: calc(100% - 30px);
+    }
+
+    #modx-login-logo {
+        margin: 0 auto 20px;
+        img {
+            margin: 0 auto;
+            display: block;
+        }
+    }
+
+    #modx-panel-login-div{
+        padding: 25px;
+    }
+
+    .login-form-element .x-form-text{
+        width: -webkit-fill-available;
+    }
+
+    .login-cb-col {
+        float: none;
+        width: 100%;
+    }
+
+    #modx-login-btn{
+        float: none;
+        margin: 20px auto 0;
+        display: block;
+    }
+
+    .x-form-check-wrap.modx-login-rm-cb{
+        float: none;
+    }
+
+    .loginLicense{
+        width: calc(100% - 30px);
+    }
```

Great work! To keep the code style consistent I suggest to add spaces between the selectors and brackets, you can apply my suggestions instantly via the GitHub webinterface. 

```suggestion
@media (max-width: 640px) {
    #container {
        margin: 50px auto 0;
        width: calc(100% - 30px);
    }

    #modx-login-logo {
        margin: 0 auto 20px;
        img {
            margin: 0 auto;
            display: block;
        }
    }

    #modx-panel-login-div {
        padding: 25px;
    }

    .login-form-element .x-form-text {
        width: -webkit-fill-available;
    }

    .login-cb-col {
        float: none;
        width: 100%;
    }

    #modx-login-btn {
        float: none;
        margin: 20px auto 0;
        display: block;
    }

    .x-form-check-wrap.modx-login-rm-cb {
        float: none;
    }

    .loginLicense {
        width: calc(100% - 30px);
    }
```